integration-tests/actions/auto_ocr_helpers.go

```diff
@@ -0,0 +1,291 @@
+package actions
+
+//revive:disable:dot-imports
+import (
+	"crypto/ed25519"
+	"encoding/hex"
+	"fmt"
+	"math"
+	"math/big"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/lib/pq"
+	. "github.com/onsi/gomega"
+	"github.com/rs/zerolog/log"
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink-testing-framework/contracts/ethereum"
+	"github.com/smartcontractkit/libocr/offchainreporting2/confighelper"
+	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+	types2 "github.com/smartcontractkit/ocr2keepers/pkg/types"
+	"github.com/umbracle/ethgo/abi"
+	"gopkg.in/guregu/null.v4"
+
+	"github.com/smartcontractkit/chainlink/core/services/job"
+	"github.com/smartcontractkit/chainlink/core/services/keystore/chaintype"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+	"github.com/smartcontractkit/chainlink/integration-tests/client"
+	"github.com/smartcontractkit/chainlink/integration-tests/contracts"
+)
+
+func BuildOCRConfigVars(chainlinkNodes []*client.Chainlink, registryConfig contracts.KeeperRegistrySettings, registrar string) contracts.OCRConfig {
+	S, oracleIdentities := getOracleIdentities(chainlinkNodes)
+
+	signerOnchainPublicKeys, transmitterAccounts, f, _, offchainConfigVersion, offchainConfig, err := confighelper.ContractSetConfigArgsForTests(
+		10*time.Second,       // deltaProgress time.Duration,
+		10*time.Second,       // deltaResend time.Duration,
+		3*time.Second,        // deltaRound time.Duration,
+		500*time.Millisecond, // deltaGrace time.Duration,
+		2*time.Second,        // deltaStage time.Duration,
+		3,                    // rMax uint8,
+		S,                    // s []int,
+		oracleIdentities,     // oracles []OracleIdentityExtra,
+		types2.OffchainConfig{
+			PerformLockoutWindow: 100 * 12 * 1000, // ~100 block lockout (on goerli)
+			UniqueReports:        false,           // set quorum requirements
+		}.Encode(), // reportingPluginConfig []byte,
+		50*time.Millisecond, // maxDurationQuery time.Duration,
+		time.Second,         // maxDurationObservation time.Duration,
+		time.Second,         // maxDurationReport time.Duration,
+		50*time.Millisecond, // maxDurationShouldAcceptFinalizedReport time.Duration,
+		50*time.Millisecond, // maxDurationShouldTransmitAcceptedReport time.Duration,
+		1,                   // f int,
+		nil,                 // onchainConfig []byte,
```

Comment:
Would these be mostly standard for all ocr2 jobs? would it make sense to pass these as a parameter leveraging [this](https://github.com/smartcontractkit/chainlink/blob/acdbe197438553b2c5effdf6b4f5ab6d8a0f09e8/integration-tests/contracts/contract_models.go#L112)?

Reply:
I will be adding more tests soon and ill look at packing this up in those